pages/room/[...id].js

```diff
@@ -97,21 +103,38 @@ const Room = () => {
         });
 
         socketRef.current.on('return message', (payload) => {
-          console.log(payload.username);
-          console.log(currentUsername);
           setConversation((prevConversation) => {
             return [
               ...prevConversation,
               { text: payload.message, sender: payload.username, fromMe: payload.username == currentUsername },
             ];
           });
         });
+
+        socketRef.current.on('user disconnect', (payload) => {
+          let usersPeerID = [];
+          let participantNames = [];
+          if (payload.users) {
+            usersPeerID = payload.users.map((user) => user.socketID);
+            participantNames = payload.users.map((user) => user.username);
+          }
+          peersRef.current.forEach((peerRef) => {
+            if (usersPeerID.length > 0 && !usersPeerID.includes(peerRef.peerID)) {
```

Comment:
you can probably make it so that if `usersPerrId.length <= 0` just exit the function

Reply:
that way you don't have to evaluate `usersPeerId.length` every iteration